No to multiple brokers for now 😢 (#1995)
semgrep/semgrep-docs: docs/semgrep-ci/network-broker.md

## Run multiple instances of the Semgrep Network Broker

> You should not attempt to run multiple instances of the Semgrep Network Broker to increase availability. Running multiple instances can result in contention and is currently less reliable than running a single instance.

Reviewer: ```suggestion
Do not attempt to run multiple instances of the Semgrep Network Broker to increase availability. Running multiple instances can result in contention and is less reliable than running a single instance.
```

> ## Allowlisting multiple source code managers with one configuration file

Author: oh of course, thanks!
